gluten-ut/spark34/src/test/scala/org/apache/spark/sql/GlutenDynamicPartitionPruningSuite.scala

```diff
@@ -550,17 +606,26 @@ class GlutenDynamicPartitionPruningV1SuiteAEOff
           scanOption.get
         }
 
+        def getDriverMetrics(plan: SparkPlan, key: String): Option[SQLMetric] = {
```

Comment:
Looks this function is duplicated several times. If any reason we need it as we always pass scan for `plan`.

Reply:
@zhli1142015 he corresponding metrics need to e sourced from driverMetrics (as against metrics, used earlier)